include/dxc/Support/dxcapi.use.h

```diff
@@ -178,6 +193,64 @@ void WriteOperationErrorsToConsole(IDxcOperationResult *pResult,
 void WriteOperationResultToConsole(IDxcOperationResult *pRewriteResult,
                                    bool outputWarnings);
 
+class DxcDllExtValidationSupport : public DxcDllSupport {
+  // DxcDllExtValidationSupport manages the
+  // lifetime of dxcompiler.dll, while the member, m_DxilSupport,
+  // manages the lifetime of dxil.dll
+  DxcDllSupport *m_DxilSupport = nullptr;
+
+  std::string DxilDLLPathExt = "";
+  bool ExternalDxilDLLInitializationSuccess = false;
```

Comment:
No need for empty initialization for `std::string`, I don't think.

We also don't need `ExternalDxilDLLInitializationSuccess` because this can be detected by: `m_DxilSupport.IsEnabled()`.

We actually want a method that returns whether we attempted to load the DLL and it failed.  That state can be detected by `!DxilDLLPathExt.empty() && !m_DxilSupport.IsEnabled()`.

Reply:
Also, I'm not sure why the `Ext` is needed in the `DxilDLLPathExt`.  Also, we should stick with the convention established with the class name on casing of dll:

 ```suggestion
  std::string DxilDllPath;
```